lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html

```diff
@@ -7,7 +7,7 @@
 
     <div *ngSwitchDefault>
         <mat-form-field class="adf-property-field adf-card-textitem-field"
-                        [ngClass]="{ 'adf-property-read-only': !isEditable }"
+                        [ngClass]="{ 'adf-property-read-only': !isEditable,'adf-property-field-has-error': isEditable && hasErrors }"
```

Comment:
make sure that this new class name is really needed.

Reply:
Yes, it is needed. Without it mat-form-field has no class indicating validation error even if such error exist (probably because form is template-driven).